docs/blog/author.yaml

```diff
@@ -157,3 +157,7 @@
   bio: Software developer and full-time opensourcerer at @contentful. He speaks multiple languages and is often overheard saying "Bonjour" in HTML.
   avatar: avatars/khaled-garbaya.jpg
   twitter: "@khaled_garbaya"
+- id: Linda Watkins
```

Comment:
You're already on here a few lines ago from your previous post -- should edit that instead of adding a new bio